packages/models/src/lib/content/folders/folder.model.ts

```diff
@@ -0,0 +1,14 @@
+import { UserInfo } from '@pulp-fiction/models/users';
+import { Types } from 'mongoose';
+
+export interface Folder {
+    readonly _id: Types.ObjectId;
```

Comment:
(should we start using this for `_id`s everywhere?)

Reply:
only where the documents we're creating aren't ones where we care about URL-friendly IDs